src/utils.test.js

```diff
@@ -664,4 +668,15 @@ describe('utils', () => {
     const nullObject = null;
     expect(omitKeys(nullObject)).toEqual(null);
   });
+
+  test('safeExecute run function', () => {
+    expect(safeExecute(() => 5)()).toEqual(5);
+  });
+
+  test('safeExecute catch exception', () => {
```

Comment:
Add test for async func

Reply:
Look at the comment of adding async support :)

Reply:
🎸